README.md

```diff
@@ -22,6 +22,57 @@ It can be used to scaffold high-quality draft genome assemblies with any long
 sequences (eg. ONT reads, PacBio reads, other draft genomes, etc).
 It is also used to scaffold contig pairs linked by ARCS/ARKS.
 
+### Dependencies
+-------------------------------
+
+- GCC (tested on v7.2.0)
+- Perl (tested on v5.32.1)
+- Autotools (if cloning directly from repository)
+
+### Compilation:
+-------------------------------
+If cloning directly from the repository run:
+
+```
+cd releases/links_v2.0/
+git submodule init
+git submodule update
+```
+
+Generate autotools scripts:
+
+```
+./autogen.sh
+```
+
+To compile LINKS run:
+
+```
+./configure && make
+```
+To install LINKS in a specified directory:
+
+```
+./configure --prefix=/LINKS/PATH && make install
+```
+
+### Running LINKS v2.0
+-------------------------------
+LINKS v2.0 is implemented in C++ and perl. To run the full LINKS pipeline smoothly we suggest running `LINKS-make`
+
+To run LINKS with default parameters with `LINKS-make`:
```

Comment:
```suggestion
To run LINKS with default parameters with `LINKS`:
```
We spoke about this before, but where did we land on it? If a user updates the version and run `LINKS`, will it just run the C++ version only? I think the ideal would be to have a wrapper script `LINKS` that could call all the steps, and still take the same parameters as previously